parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -247,6 +247,9 @@ else
 fi
 
 if [[ "${MESSAGE_OF_THE_DAY}" != "" ]]; then
+    if isMarinerOrAzureLinux "$OS" && [ -f /etc/dnf/automatic.conf ]; then
+      sed -i "s/emit_via = motd/emit_via = stdio/g" /etc/dnf/automatic.conf
```

Comment:
where does it go when `stdio` is used ?

Reply:
Context: https://dnf.readthedocs.io/en/latest/automatic.html 
just to standard output, i believe this particualr part of dnf is triggered by a systemd unit so that would go into journald